improved_note_detection.py

```
@@ -0,0 +1,77 @@
+This script was added as a quick experiment and isn't part of the main
+``oemer`` package.  The logic attempts to clean up the input score image
+using a few morphological operations and then extracts note heads by
+finding contours with a roughly circular aspect ratio.  It still relies on
+``cv2`` and ``numpy`` at runtime.
+"""
+
+import cv2
+import numpy as np
+
+
+def detect_notes(image_path: str):
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)